Tribler/Test/test_as_server.py

```diff
@@ -125,6 +125,11 @@ def tearDown(self, annotate=True):
         self.assertFalse(delayed_calls, "The reactor was dirty when tearing down the test")
         self.assertFalse(Session.has_instance(), 'A session instance is still present when tearing down the test')
 
+        # Clean the delayed calls to prevent them cascading into the next test case
+        for delayed_call in reactor.getDelayedCalls():
```

Comment:
Really? I just ran the /Core test suite just fine. But what you probably are referring to is that there needs to be an instance with actually a delayed call which probably did not happen because all tests passed.